### PR TITLE
Change README.md formatting

### PR DESCRIPTION
**Pull request by aureooms**
* Use `h3 (###)` instead of bullet points
* Organize install steps into larger chunks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,6 +265,7 @@ If you want `coauthor@yourhostname.com` to receive email,
 add an alias like `coauthor: edemaine@mit.edu` to `/etc/aliases`
 and then run `sudo newaliases`.
 
+## Performance monitoring ###
```
Comment (aureooms) [inline README.md]: Of course this needs to be updated at some point (https://github.com/edemaine/coauthor/issues/204), just changing the style for the moment.

File: README.md [hunk 2]
```diff
@@ -226,9 +250,10 @@ To deploy to a public server, we recommend
 3. `mup deploy` each time you want to deploy code to server
```
Comment (aureooms) [inline README.md]: Don't you need to run those commands inside the `.deploy` directory?

File: README.md [hunk 3]
```diff
@@ -6,171 +6,190 @@ collaboration on unsolved problems in theoretical computer science, so
...
-  including a not-yet-visible reparenting feature.
-
-* **Superuser operations** (only for superusers):
-  * Import from LaTeX document with figures attached as a ZIP file
-  * Import from osqa's XML dump, including old edit history
-  * Superdelete (permanently destroying a message including its history)
-  * Setting the default sort for a group
+## Features so far ##
+
+### Live updates ###
```
Comment (aureooms) [inline README.md]: Allows to point to documentation sections via `#{title}` e.g. `#live-updates` or `#groups`.

File: README.md (hunk 1 above)
Comment (edemaine) [inline README.md]: Updated (though I still haven't tried open-source Kadira myself).

File: README.md (hunk 2 above)
Comment (edemaine) [inline README.md]: Added `cd .deploy` step

**Comment by edemaine** [comment]
Sorry for the delay in reviewing this, and thanks for the suggestions!

As much as I generally like sections, I worry that this will make the front page too long.  Perhaps a split into high-level and more detailed documentation is in order (especially if we add screenshots to go with the features, which is probably a good idea).  But for now I'm going to pass on the section suggestions (though I appreciate the deep linking point).  I've implemented the other comments, though -- thanks!

**Pull request closed without merging**